kore/src/Kore/Log.hs

```diff
@@ -208,12 +210,16 @@ makeKoreLogger exeName startTime timestampSwitch logActionText =
                         toMicroSecs (diffTimeSpec startTime entryTime)
         toMicroSecs = (`div` 1000) . toNanoSecs
     exeName' = Pretty.pretty exeName <> Pretty.colon
-    prettyActualEntry timestamp ActualEntry{actualEntry, entryContext} =
-        (Pretty.vsep . concat)
-            [ [header]
-            , indent <$> context'
-            , indent <$> [longDoc actualEntry]
-            ]
+    prettyActualEntry timestamp ActualEntry{actualEntry, entryContext}
+        | OneLine <- koreLogFormat =
+            Pretty.hsep $
+                catMaybes [timestamp, Just (Pretty.parens $ type' actualEntry), oneLineDoc actualEntry]
```

Comment:
I'm inclined to add the `severity` information as well, since users can filter logs depending on just that.